crates/storage/src/tables/genesis_time.rs

```diff
@@ -0,0 +1,40 @@
+use std::sync::Arc;
+
+use redb::{Database, Durability, TableDefinition};
+
+use super::{Field, SSZEncoding};
+use crate::errors::StoreError;
+
+/// Table definition for the Genesis_Time table
+///
+/// Value: u64
+pub const GENESIS_TIME_FIELD: TableDefinition<&str, SSZEncoding<u64>> =
```

Comment:
```suggestion
pub const GENESIS_TIME_FIELD: TableDefinition<&str, u64> =
```

u64 doesn't need to be wrapped in SSZEncoding